src/asm_parse.cpp

```diff
@@ -68,6 +68,8 @@ static const std::map<std::string, Un::Op> str_to_unop = {
     {"le32", Un::Op::LE32},
     {"le64", Un::Op::LE64},
     {"-", Un::Op::NEG},
+    {"zext32", Un::Op::ZEXT32},
+    {"sext32", Un::Op::SEXT32},
```

Comment:
there are multiple new signed extension instructions that are Bin ops (not Un ops), so I don't think there's any reason to add Un ops here.

Reply:
I think sext32 can now be done using the MOVSX instruction